.github/workflows/flutter-test.yml

```
@@ -25,6 +25,8 @@ jobs:
       - run: echo "::add-path::$GITHUB_WORKSPACE/_flutter/bin"
       - run: flutter pub get
         working-directory: ./client/flutter
+      - run: flutter format --set-exit-if-changed --dry-run lib/
```

```suggestion
      - run: flutter format . --set-exit-if-changed --dry-run --fix
```

I think that `.` is better because it will also format the `test` directory and potentially `test_driver` in the future.